contracts/tokens/Cruzo1155.sol

```diff
@@ -109,7 +116,7 @@ contract Cruzo1155 is ERC1155URI {
         bytes memory _data
     ) public onlyCreator(_tokenId) returns (uint256) {
         require(
```

Comment:
This is an unreachable check, it's already checked in the `onlyCreator` modifier